doc/1.4/language.md

```diff
@@ -2332,12 +2332,34 @@ So the two following declarations are equivalent:
 }
 ```
 
-is equivalent to
-
 ```
 register R size 4;
 ```
 
+As a special exception, an `#if` statement that appears on top level is allowed
+to contain any type of statement, as long as the condition doesn't reference
+any identifiers other than `dml_1_2`, `true` and `false`. This is often useful
+while migrating the devices of a system from DML 1.2 to DML 1.4,
+as it allows conditional definitions of templates in common code used from both DML 1.2 and DML 1.4. For example, let's say an existing template `reset_to_seven`
```

Comment:
line surpasses 79 char width limit

Reply:
I don't think the same style rule applies for .md; IIRC @aconradi suggested "new line between sentences" as an alternative convention to make diffs more manageable.

For markdown embedded in source comments, the host language's conventions take precedence of course.

Reply:
Whichever criteria you choose, you fail it. If nothing else, "For example," should be on a new line.

I still vote for breaking on spaces _except_ when \` is involved, because apparently the GH wiki inteprets line breaks within tick literally (which we need to clean up...)